### PR TITLE
Draft pull request template

### PR DESCRIPTION
**Pull request by tristen**

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+## Description of change
+
+<!-- Briefly describe your changes here. If you are replacing an existing icon, provide a before and after screenshot -->
+
+### Link to demonstration
+
+<!-- Provide a link to a interactive map that uses the new icon with 11x11 and 15x15 variation. -->
```
Comment (vknoppkewetzel) [inline .github/pull_request_template.md:7]: I know we already chatted about this in the meeting, but removing the 11x11 and 15x15 here, and then updating to what we ultimately decide on (I am not sure if we agreed on 18x18 or 24x24 on Friday)

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+---
+
+Closes #<!--insert issue number here-->.
+
+### For author
+
+- [ ] Cite changes under the `HEAD` tag in [`CHANGELOG.md`](../CHANGELOG.md)
+- [ ] PR contains only one icon with 11x11 and 15x15 variation
```
Comment (vknoppkewetzel) [inline .github/pull_request_template.md:16]: remove 11x11 / 15x15

**Review by vknoppkewetzel — CHANGES_REQUESTED** [review]
This looks great Tristen! Really just added what was already verbally discussed on Friday, but wanted to make sure that was written down. 

Reading this, also thought about how this would be useful for not just maki, but other icon related PR work (hw shields, for example). Once this is all approved, would love to take this to MCSI

**Review by ClareTrainor — APPROVED** [review]
👍 looks good!

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+- [ ] Changes fall in accordance to the [Maki guidelines](https://labs.mapbox.com/maki-icons/guidelines/)
```
Comment (willymaps) [inline .github/pull_request_template.md:17]: Should we create a new guidelines page? Or an internal page for dev?

File: .github/pull_request_template.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+  - [ ] Icon geometry is aligned to the pixel grid
+  - [ ] Icon stays within a 2px trim area
```
Comment (willymaps) [inline .github/pull_request_template.md:19]: Not sure where we landed on trim area. Is this still true?

Comment (tristen) [reply .github/pull_request_template.md:19]: I think a _trim area_ is still true. for background fills we want padding space. Where we land on pixel dimension is still up for debate?

**Review by kelsey-taylor — APPROVED** [review]
No changes beyond the comments others have left already! Thanks for updating this @tristen 🐙 

One item you could include for the reviewer is to upload new icons to the [Maki icon editor](https://labs.mapbox.com/maki-icons/editor/) 1) for comparison with existing icons and 2) to test how stroke/backgrounds/padding look in various styles.

File: .github/pull_request_template.md (hunk 4 above)
Comment (willymaps) [reply .github/pull_request_template.md:19]: True, I guess we can adjust as we experiment with dimensions. Thanks!

**Review by willymaps — APPROVED** [review]
Thanks for putting together @tristen

**Review by vknoppkewetzel — APPROVED** [review]
:shipit: looks great!

**Pull request merged**